### DISCUSSION TITLE
Show/hide a column depending on the state of the record inside a relation manager table

### DISCUSSION THREAD

**Discussion by maxacarvalho** [filamentphp/filament#7411] [Help]
Hi!

I'm trying to add a column type depending on the state of the record inside a relation manager table.

My use case requires me to check if the relation item is in a certain status. Depending on that value I have to render a text column or an input text column.

Thanks.

**Comment by danharrin** [comment]
Unfortunately you cannot mix column types within a single column

**Comment by maxacarvalho** [comment]
Hi @danharrin 
Oh yeah, that makes sense now that I thought about it.

Thanks.
